package.json

```diff
@@ -28,7 +28,9 @@
   "bin": {
     "gsn-stat": "./scripts/gsn-stat.js",
     "gsn-dock-relay": "./scripts/gsn-dock-relay",
-    "gsn-dock-relay-ganache": "./scripts/gsn-dock-relay-ganache"
+    "gsn-dock-relay-ganache": "./scripts/gsn-dock-relay-ganache",
```

Comment:
Maybe we don't need to pupulate bin with docker commands? This feels excessive to me, tbh.

Reply:
I didn't work on removing docker scripts from here, maybe we can remove it.

Reply:
removed